File: 프로그래머스/0/120834. 외계행성의 나이/외계행성의 나이.js
function solution(age) {
    var answer = '';
    const ageProgram = ['a','b','c','d','e','f','g','h','i','j'];
    const strAge = String(age);
    for(let i=0; i<strAge.length; i++){
        const index = parseInt(strAge[i]);
        answer += ageProgram[index];
    }
    return answer;
}

File: 프로그래머스/0/120834. 외계행성의 나이/외계행성의 나이.py
def solution(age):
    answer = ''
    alpha = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j']
    age = [int(i) for i in str(age)]
    for j in age:
        answer += alpha[j]
    return answer